docs/release-notes/2.0.0.asciidoc

```diff
@@ -0,0 +1,76 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-2.0.0]]
+== {n} version 2.0.0
+
+[[breaking-2.0.0]]
+[float]
+=== Breaking changes
+
+* Remove legacy v1beta1 CRDs {pull}5029[#5029] (issue: {issue}4885[#4885])
+
+
+[[feature-2.0.0]]
+[float]
+=== New features
+
+* Allow K8S node labels to be propagated as Pod annotations {pull}5054[#5054] (issue: {issue}3933[#3933])
+
+[[enhancement-2.0.0]]
+[float]
+=== Enhancements
+
+* Allow users to disable version validation to downgrade clusters  {pull}5272[#5272] (issue: {issue}5259[#5259])
+* Add support for transport TLS certificate other/common name suffix {pull}5189[#5189] (issue: {issue}5148[#5148])
+* Bump the operator memory limit to 1Gi for larger deployments {pull}5185[#5185] (issue: {issue}5177[#5177])
+* Move config hash stored in Pod labels to Pod annotations {pull}5175[#5175] (issue: {issue}5071[#5071])
```

Comment:
Should we also update https://github.com/elastic/cloud-on-k8s/blob/main/docs/operating-eck/upgrading-eck.asciidoc#control-rolling-restarts-during-the-upgrade ?

Reply:
I have gone over the upgrade page and also removed the bits referring to the older versions of Kubernetes we no longer support.